src/utils/react.js

```diff
@@ -18,3 +18,13 @@ export let expandProp = _.flow(
   (key, fn) => F.expandObjectBy(key, F.whenExists(fn)),
   mapProps
 )
+
+export let defaultProps = props => BaseComponent => {
```

Comment:
why didn't we just use [`defaultProps` from `recompose`](https://github.com/acdlite/recompose/blob/master/docs/API.md#defaultprops)?